docs/en/observability/monitor-uptime-synthetics.asciidoc

```diff
@@ -5,9 +5,20 @@
 <titleabbrev>Synthetic monitoring</titleabbrev>
 ++++
 
-The {uptime-app} in {kib} lets you monitor user journeys and full page loads using our browser monitor,
-and also monitor the availability and response times of applications and services using our lightweight HTTP/S,
-TCP, and ICMP monitors.
+Synthetic monitors go beyond classic end-to-end tests and monitoring practices to ensure that your software continues to work as expected in many conditions. 
+
+Even the most thorough end-to-end tests can't prove your software works because you can't validate all possible states of your application.
```

Comment:
This phrase might be confusing for users outside of the context of a story or a concrete example, so I'd probably find a way to make this more concrete, i.e.:

> By using synthetic monitors, you can detect bugs caused by invalid states you couldn't foresee, and thus didn't write tests for. Furthermore, synthetic monitors help you catch bugs in features that don't get enough traffic by allowing you to periodically simulate users' actions.